scripts/generate-changelog.sh

```diff
@@ -1,10 +1,12 @@
 #!/bin/bash
 
 REPO_DIR=$(git rev-parse --show-toplevel)
+RELEASE=v$(grep '^DISTRO_VERSION ' "$REPO_DIR/meta-dts-distro/conf/distro/dts-distro.conf" | cut -d'=' -f2 | tr -d ' "')
```

Comment:
Will this create e.g. `## [v2.1.2]` header? Because currently all of our headers use `## [2.1.2]` format (without `v`)
Maybe we should fix previous headers to be consistent?

Reply:
No, it will omit the `v`